src/style.css

```diff
@@ -3,6 +3,79 @@
 @tailwind utilities;
 
 
-:root {
+/*
```

Comment:
You are using Tailwind utilities, so you shouldn't need additional styling below. Adding those styles can potentially conflict with Tailwind's utility classes and cause unintended results.

Tailwind provides a set of pre-defined utility classes that you can use to style your components without writing any CSS.